src/items/dto/draft-item-request.dto.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  IsNotEmpty,
+  IsPositive,
+  MaxLength,
+  Min,
+  Max,
+  IsEthereumAddress,
+  MinLength,
+  IsArray,
+  ArrayMinSize,
+  ArrayMaxSize,
+  ArrayNotEmpty,
+} from 'class-validator';
+
+export class DraftItemRequestDto {
+  @IsEthereumAddress()
+  @IsNotEmpty()
+  address: string;
+
+  @IsEthereumAddress()
+  @IsNotEmpty()
+  collectionAddress: string;
+
+  @IsNotEmpty()
+  @MinLength(1)
+  @MaxLength(100)
+  name: string;
+
+  @IsNotEmpty()
+  description: string;
+
+  @Min(0)
+  @Max(10)
+  @IsPositive()
+  royalty: number;
+
+  @IsArray()
+  @ArrayNotEmpty()
```

Comment:
I have a question. If you put a minimun length why do you put a Empty condition?